docs/userGuide/cliCommands.md

```diff
@@ -30,9 +30,9 @@ MarkBind Command Line Interface (CLI) can be run in the following ways:
 
 ### `build` Command
 
-**Format:**: `markbind build [<OPTIONS>] `
+**Format:** `markbind [build | b] [<OPTIONS>] `
```

Comment:
Currently we use square brackets to mean optional. The commands themselves aren't optional, so maybe use `<build | b>` instead? Or drop the brackets altogether: `build|b` (which is how `markbind --help` renders).

Or an alternative would be to specify it on a separate line? Something similar to [how npm docs does it](https://docs.npmjs.com/cli/install):

```markdown
**Format:** `markbind build [OPTIONS]`
**Alias:** `markbind b`
```

Reply:
Ah, thanks for clarifying. I like how it looks on a separate line - just pushed the changes for that 🙂